testing/sync-test/src/auth.rs

```diff
@@ -109,11 +57,38 @@ impl TestAccount {
         pass: String,
         cfg: FxaConfig,
         no_delete: bool,
-    ) -> Result<Arc<TestAccount>, failure::Error> {
+    ) -> Result<Arc<TestAccount>> {
         log::info!("Creating temporary fx account");
-        // `create` doesn't return anything we care about.
-        let auth_url = cfg.auth_url()?;
-        run_helper_command("create", &[&email, &pass, auth_url.as_str()])?;
+
+        restmail::clear_mailbox(&email);
+
+        let create_endpoint = cfg.auth_url_path("v1/account/create").unwrap();
```

Comment:
You may be able to simplify things here, by fetching the sync keys during account creation. If you specify `?keys=true` in the URL here, then it will return a `sessionToken` and `keyFetchToken` that you can use in the same way as the ones you get in `account_login`. You could verify the account like you do here already, then fetch the keys and store them as a property on the `TestAccount`.

Later, when you can to execute the oauth flow, you wouldn't need to log in to the account again, you could just use the keys and sessionToken obtained during account creation.